forte/data/data_store.py

```diff
@@ -1857,8 +2081,18 @@ def _parse_onto_file(self):
             entry_name = entry_node.name
             if entry_name in DataStore._type_attributes:
                 continue
+
+            entry_dict = {}
```

Comment:
This is a duplicate implementation of line  2110 - 2112.

Reply:
I think the only difference would be that `_init_top_to_core_entries`will add the parent class to `type+attributes immediately. But I can change the code to incorporate it.

Reply:
I mean these lines:
```python
entry_dict = {}
entry_dict[constants.PARENT_CLASS_KEY] = set()
entry_dict[constants.PARENT_CLASS_KEY].add(entry_node.parent.name)
```
Why do we need to call them twice?

Reply:
Oh I got you. I'll fix it